### DISCUSSION TITLE
"Stealing" an IP from a VPS for a Home Server

### DISCUSSION THREAD

**Discussion by rzvend** [wg-easy/wg-easy#476] [General]
Hi, I would like if it is possible to use Wg-Easy to set up the Wireguard link between a VPS and my home server to expose service from home to the internet using Nginx Proxy Manager.

I already tried the Wg-Easy to set up the VPN and it is working, but I still have not figured out how to set up this scenario using the VPS + Wireguard + Nginx Proxy Manager. I am behind a CGNAT I am using Cloudflare Tunnel for this purpose but it would be great to have this alternative.

I am already thankful for any help

Ricardo

**Comment by raihangit000** [comment]
also looking for same solution.
